target/linux/lantiq/files-5.4/arch/mips/boot/dts/lantiq/xrx330_dlink_dwr-966.dts

```diff
@@ -0,0 +1,295 @@
+// SPDX-License-Identifier: (GPL-2.0+ OR MIT)
+/dts-v1/;
+
+#include "xrx330.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,dwr-966", "lantiq,xway", "lantiq,grx390";
+	model = "D-Link DWR-966";
+
+	chosen {
+		bootargs = "console=ttyLTQ0,115200";
+	};
+
+	memory@0 {
+		reg = <0x0 0x8000000>;
+	};
+
+	aliases {
+		led-boot = &led_power_error;
+		led-failsafe = &led_power_error;
+		led-running = &led_power;
+		led-upgrade = &led_power_error;
+
+		led-internet = &led_internet;
+		led-wifi = &led_wifi_2g;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <100>;
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 9 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+		wps {
+			label = "wps";
+			gpios = <&gpio 11 GPIO_ACTIVE_HIGH>;
+			linux,code = <KEY_WPS_BUTTON>;
+		};
+		ant {
+			label = "ant";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			linux,code = <BTN_0>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wps {
+			label = "dwr-966:green:wps";
```

Comment:
The model prefix (dwr-966:) has been dropped from LED labels, so you can just use "green:wps" etc.